Common/src/CConfig.cpp

```diff
@@ -3395,9 +3395,15 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
     SU2_MPI::Error("A turbulence model must be specified with KIND_TURB_MODEL if SOLVER= INC_RANS", CURRENT_FUNCTION);
   }
 
+  /*--- Postprocess SST_OPTIONS into structure. ---*/
+  if (Kind_Turb_Model==TURB_MODEL::SST){
+    sstParsedOptions = ParseSSTOptions(SST_Options, nSST_Options, rank);
+    using_uq = sstParsedOptions.uq;
```

Comment:
I guess this is where the uq bool comes into play.  Would it be better to do:
`using_uq = sstParsedOption.prouduction == SST_OPTIONS::UQ` to avoid multiple paths of data?

Reply:
It would be better to rely only on the struct throughout the code, no extra booleans.

Reply:
ok, so we remove using_uq

Reply:
I have kept config::getUsingUq() but it now returns sstParsedOptions.uq, it is only used in viscous_fluxes.hpp.